research-software-directory/RSD-as-a-service
# fix: enable save button on inital person import

# Enable save button on initial import

Closes #1030

Changes proposed in this pull request:
* When importing contributors the Save button is enabled without any editing   
* When importing team member the Save button is enabled without any editing   
* If there are avatar images, the "initials" avatar is positioned after the images and the first image is (pre)selected

How to test:
* `make start` to build app and generate data
* login as rsd_admin to edit existing software/projects or create new software and project
* Edit software contributors. Add "Dusan Mijatovic" for example. Confirm that Save button is enabled
* Add avatar image
* Navigate to another software and add Dusan Mijatovic again. Confirm that Save button is enabled and that avatar image is selected by default.
* Navigate to edit project team members. Add Dusan Mijatovic. Confirm that Save button is enabled and image is selected. Add another avatar image.
* Edit now another project/software item. Add Dusan Mijatovic. Confirm that Save button is enabled and avatar image is  (pre)selected. Try changing avatar.

PR Checklist:
